apollo-federation/src/link/argument.rs

```diff
@@ -100,22 +100,6 @@ pub(crate) fn directive_optional_boolean_argument(
     }
 }
 
-#[allow(dead_code)]
-pub(crate) fn directive_required_boolean_argument(
-    application: &Node<Directive>,
-    name: &Name,
-) -> Result<bool, FederationError> {
-    directive_optional_boolean_argument(application, name)?.ok_or_else(|| {
-        SingleFederationError::Internal {
-            message: format!(
-                "Required argument \"{}\" of directive \"@{}\" was not present.",
-                name, application.name
-            ),
-        }
-        .into()
-    })
-}
-
```

Comment:
This was not part of the ticket, but I stumbled across this dead code that was not clean up during the `@context` PR, so I removed it.